src/components/WalletModal/index.tsx

```diff
@@ -369,14 +372,52 @@ export default function WalletModal({
       )
     } else {
       headerRow = (
-        <HeaderRow>
+        <HeaderRow redesignFlag={redesignFlagEnabled}>
           <HoverText>
             <Trans>Connect a wallet</Trans>
           </HoverText>
         </HeaderRow>
       )
     }
 
+    function getTermsOfService(redesignFlagEnabled: boolean) {
+      return redesignFlagEnabled ? (
+        <AutoRow style={{ flexWrap: 'nowrap', padding: '4px 16px' }}>
+          <ThemedText.BodySecondary fontSize={12}>
+            <Trans>
+              By connecting a wallet, you agree to Uniswap Labs’{' '}
+              <ExternalLink style={{ textDecoration: 'underline' }} href="https://uniswap.org/terms-of-service/">
+                Terms of Service
+              </ExternalLink>{' '}
+              and acknowledge that you have read and understand the Uniswap{' '}
```

Comment:
rm {' '}

Reply:
(same for above on 388 and 391)

Reply:
this extra space is necessary in front of the links or else it looks squished: 
![Screen Shot 2022-08-17 at 12 56 41 PM](https://user-images.githubusercontent.com/41491154/185198579-29d591aa-869e-4548-b5ea-c59ae2d3b1e3.png)